update-changelog.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
```

Comment:
This is a bug on the simple-release repo. I'll clean this up.

Reply:
Address issue at the source: https://github.com/rocajuanma/simple-release/pull/21